include/qpid/dispatch/log.h

```diff
@@ -56,6 +56,7 @@ typedef enum {
     LOG_HTTP_ADAPTOR,
     LOG_FLOW_LOG,
     LOG_ADDRESS_WATCH,
+    LOG_HTTP_OBSERVER,
```

Comment:
Should this be LOG_HTTP1_OBSERVER so I can have a LOG_HTTP2_OBSERVER ?

Reply:
Sure I'll make that change - update to follow...

Reply:
Done